index.html

```diff
@@ -0,0 +1,33 @@
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio project two</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <section class="home">
+      <div class="header">
+        <h1>Welcome</h1>
+        <a href="" class="menu"><i class="fas fa-bars"></i></a>    
+      </div>
+      <div class="headline">
+        <h1>Hey There. I’m Aster</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.</p>
```

Comment:
- [ ] In order to respect the design, kindly, remove italic style applied to the paragraph;